templates/init/src/js/index.js

```diff
@@ -39,7 +39,7 @@ var Main = React.createClass({
   },
   render: function() {
     return (
-      <App centered={false}>
+      <App centered={false} lang={'en'}>
```

Comment:
Do we want to fix that to `en` ? I would leave this empty and use the browser settings instead.